tools/tests/old/test-watch.js

```diff
@@ -1,3 +1,5 @@
+require('babel/register')(require('../../babel-config.js'));
```

Comment:
This repetition is unfortunate. Does code in these old test files need to be transpiled at all?